packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1910,3 +1923,437 @@ async fn nested_enums_are_correctly_encoded_decoded() {
 
     assert_eq!(response.value, expected_none);
 }
+
+#[tokio::test]
+async fn can_call_no_arg_predicate_returns_true() {
+    let amount_to_predicate: u64 = 16;
+    let asset_id = AssetId::default();
+    let predicate_code =
+        std::fs::read("tests/test_projects/predicate_true/out/debug/predicate_true.bin").unwrap();
+    let receiver_address: Address = LocalWallet::new_random(None).address();
+    let mut wallet = LocalWallet::new_random(None);
+    let coins = setup_single_asset_coins(
+        wallet.address(),
+        BASE_ASSET_ID,
+        DEFAULT_NUM_COINS,
+        DEFAULT_COIN_AMOUNT,
+    );
+    let (_launched_provider, address) = setup_test_provider(
+        coins,
+        Some(Config {
+            predicates: true,
+            utxo_validation: true,
+            ..Config::local_node()
+        }),
+    )
+    .await;
+    let connected_provider = Provider::connect(address).await.unwrap();
+    wallet.set_provider(connected_provider);
+
+    let instance = Predicate::new(predicate_code);
+    instance
+        .create_predicate(&wallet, amount_to_predicate, asset_id)
+        .await
+        .unwrap();
+    let receiver_balance_before = get_balance(&wallet, receiver_address, asset_id).await;
+    assert_eq!(receiver_balance_before, 0);
+    instance
+        .spend_predicate(
+            &wallet,
+            amount_to_predicate,
+            asset_id,
+            receiver_address,
+            None,
+        )
+        .await
+        .unwrap();
```

Comment:
Use `?` here too, even in tests. For this, you will have to change the return type of the test to `Result<(), Error>`, see examples above in this file.